src/main.rs

```diff
@@ -20,6 +22,17 @@ fn main() -> Result<()> {
     }
     env_logger::init();
 
+    StdErr::deprecation_warning(&format!(
+        r#"{} {} {}
+The version of Wrangler you are using is now deprecated.
+Please update to the latest version of wrangler to prevent critical errors.
+Run `npm uninstall -g @cloudflare/wrangler && npm install -g wrangler` to update to the latest version.
```

Comment:
They might have installed it via cargo, no?

Reply:
I will add the Cargo bit back.

Reply:
@lrapoport-cf Might need a different message template for this part.

Reply:
@JacobMGEvans @rozenmd how about the following:

```suggestion
If originally installed via npm, run `npm uninstall -g @cloudflare/wrangler && npm install -g wrangler` to update to the latest version.
If originally installed via cargo, run [xyz instructions]
```

Reply:
I am ok with this messaging, I will figure out the related `cargo` to Wrangler2 steps

Reply:
@rozenmd I updated the message